examples/form/src/schema.ts

```diff
@@ -18,19 +18,27 @@ export const BubbleTeaBaseTeaTypes = [
 export const ListOfBubbleTeaAddOns = co.list(z.literal(BubbleTeaAddOnTypes));
 export type ListOfBubbleTeaAddOns = co.loaded<typeof ListOfBubbleTeaAddOns>;
 
-export const BubbleTeaOrder = co.map({
-  baseTea: z.literal([...BubbleTeaBaseTeaTypes]),
-  addOns: ListOfBubbleTeaAddOns,
-  deliveryDate: z.date(),
-  withMilk: z.boolean(),
-  instructions: co.plainText(),
-});
+export const BubbleTeaOrder = co
+  .map({
+    baseTea: z.literal([...BubbleTeaBaseTeaTypes]),
+    addOns: ListOfBubbleTeaAddOns,
+    deliveryDate: z.date(),
+    withMilk: z.boolean(),
+    instructions: co.plainText(),
+  })
+  .resolved({
+    addOns: true,
+    instructions: true,
+  });
 export type BubbleTeaOrder = co.loaded<typeof BubbleTeaOrder>;
 
 export const PartialBubbleTeaOrder = BubbleTeaOrder.partial({
   baseTea: true,
   deliveryDate: true,
   withMilk: true,
+}).resolved({
```

Comment:
Right now `partial` and `pick` don't preserve the schema's resolve query. I don't think we have enough usage examples to decide if it's best to preserve them or not, so for now I'd prefer we don't